jobqueue/scheduler/local.go

```diff
@@ -183,8 +185,19 @@ func (s *local) initialize(config interface{}, logger log15.Logger) error {
 }
 
 // reserveTimeout achieves the aims of ReserveTimeout().
-func (s *local) reserveTimeout() int {
-	return localReserveTimeout
+func (s *local) reserveTimeout(req *Requirements) int {
+	if val, defined := req.Other["rtimeout"]; defined {
+		timeout, err := strconv.Atoi(val)
+		if err != nil {
+			s.Logger.Error(fmt.Sprintf("Failed to convert rtimeout to integer: %s", err))
+			return localReserveTimeout
+		}
+		s.Logger.Debug(fmt.Sprintf("setting runner rtimeout to %v", timeout))
```

Comment:
Go through all your files to remove explicitly referencing the embedded logger, and to not have Sprintf inside.